tests/dummy/app/templates/docs/molecules/nypr-m-box-banner.md

```diff
@@ -0,0 +1,63 @@
+## `nypr-m-box-banner`
+
+A simple box to display notifications or callouts. Includes areas for `title`, `dek`, and a `cta` button, plus an optional `close` button that will call a passed in `@close` argument.
+
+### Examples
+
+<DocsDemo as |demo|>
+
+  <demo.example>
+
+    <fieldset>
+      <legend>Show Box Banner Variations</legend>
+      <p>
+        <label for="body">Show the box content</label>
+        <input id="body" type="checkbox" onclick={{action (mut showBody) (not showBody)}}>
+      </p>
+
+      <p>
+        <label for="close">Show close button</label>
+        <input id="close" type="checkbox" onclick={{action (mut close) (action (mut close) undefined)}} checked={{close}}>
```

Comment:
not the biggest issue, but in the demo/dummy app it looks like you can't uncheck the close button